Sources/Action/Action.swift

```diff
@@ -7,11 +7,10 @@ public typealias CocoaAction = Action<Void, Void>
 /// Typealias for actions with work factory returns `Completable`.
 public typealias CompletableAction<Input> = Action<Input, Never>
 
-/// Possible errors from invoking execute()
-public enum ActionError: Error {
-    case notEnabled
-    case underlyingError(Error)
-}
+///// Possible errors from invoking execute()
+public let ActionDisabledError: Error = NSError(domain: "com.Action.Error",
```

Comment:
I am more in favor of your other suggestion in the comments, using `Action.disabledError`

Reply:
`static let` doesn't fit well with Swift Generics. Do you think this will be OK:
```
public static var DisabledError: Error { return NSError(domain: "com.Action.Error"...) }
```
Then using it as `CocoaAction.DisabledError` or `Action<Void, Void>.DisabledError`(in ActionTest)